macq/extract/esam.py

```diff
@@ -25,11 +27,14 @@ def make_PHashFluent_set(action: Action, flu: Fluent, action_2_sort: dict[str, l
     action = act(o1,o1)
     flu= lit(O1)
     output = {('lit', [object], [1]), ('lit', [object], [2])}
+
+    Parameters
+    ----------
```

Comment:
The examples should be indented and the output should be clearer on what it means